model/riscv_pmp_regs.sail

```diff
@@ -163,30 +164,29 @@ function pmpReadCfgReg(n) = {
 
 /* Helpers to handle locked entries */
 function pmpLockBit(cfg: Pmpcfg_ent) -> bool =
-   cfg.L() == 0b1
+  cfg.L() == 0b1
```

Comment:
For this commit can you make it just about adding 0x757 and split the formatting changes into a different commit to make it easier to review?